autogpt/memory/milvus.py

```diff
@@ -24,12 +24,18 @@ def __init__(self, cfg: Config) -> None:
             connect_kwargs["user"] = self.username
             connect_kwargs["password"] = self.password
 
-        connections.connect(
-            **connect_kwargs,
-            uri=self.uri or "",
-            address=self.address or "",
-            secure=self.secure,
-        )
+        if cfg.milvus_lite == "True":
```

Comment:
changes for autogpt/config/config.py is seemed to be missing.

self.milvus_lite = os.getenv("MILVUS_LITE") == "True"

this line should be in autogpt/config/config.py